library/src/main/java/com/chuckerteam/chucker/api/internal/support/SQLiteUtils.java

```diff
@@ -47,8 +47,8 @@ private static String extractDatabase(Context context) {
             File external = context.getExternalFilesDir(null);
             File data = Environment.getDataDirectory();
             if (external != null && external.canWrite()) {
-                String dataDBPath = "data/" + context.getPackageName() + "/databases/chuck.db";
-                String extractDBPath = "chuckdb.temp";
+                String dataDBPath = "data/" + context.getPackageName() + "/databases/chucker.db";
+                String extractDBPath = "chuckerdb.temp";
```

Comment:
Do we still use this method? `extractDatabase`

Reply:
Not really, I'm removing that class. You'll find it in a commit in this PR.